<?xml version="1.0" ?>
<launch> 
    <node name="localization_white_line" pkg="vision" type="localization_white_line" respawn="true" respawn_delay="5"/>
    <!--<node name="imu_3d" pkg="imu_3d" type="imu_3d" output="screen" />-->
    <node name="imu_3d" pkg="imu_3d" type="imu_3d" />
    <node name="self_localization" pkg="self_localization" type="self_localization" output="screen" respawn_delay="2"/>
</launch>
